<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OOStandartMusteriBilgileriFormu</class>
 <widget class="QDialog" name="OOStandartMusteriBilgileriFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font: 10pt &quot;Cambria&quot;;
}

QLabel#lblBaslik {
 font: bold 14pt &quot;Cambria&quot;;
}

QLabel#lblSimge {
 qproperty-pixmap: url(:/simgeler/Simgeler/standartmusteri.png);
}

QDateEdit {
 qproperty-displayFormat: 'dd.MM.yyyy';
 qproperty-calendarPopup: true;
}

QPushButton#btnKaydet {
 qproperty-text: 'Kaydet';
 qproperty-icon: url(:/simgeler/Simgeler/kaydet.png);
 qproperty-default: true;
}

QPushButton#btnTemizle {
 qproperty-text: 'Temizle';
 qproperty-icon: url(:/simgeler/Simgeler/temizle.png);
}

QPushButton#btnIptal {
 qproperty-text: 'İptal';
 qproperty-icon: url(:/simgeler/Simgeler/iptal.png);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblSimge">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblBaslik">
       <property name="text">
        <string>STANDART MÜŞTERİ BİLGİLERİ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="leTelefon"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblEposta">
       <property name="text">
        <string>E posta</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="leEposta"/>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QLineEdit" name="leInstagram"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblinstagram">
       <property name="text">
        <string>Resmi İnstagram Sayfası</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblTCNo">
       <property name="text">
        <string>TC Numarası</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblAdres">
       <property name="text">
        <string>Adres</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QPlainTextEdit" name="pteAdres">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblTelefon">
       <property name="text">
        <string>Telefon</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="leAdSoyad"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="leTCNo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblAdSoyad">
       <property name="text">
        <string>Adı Soyadı</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblDogumTarihi">
       <property name="text">
        <string>Doğum Tarihi</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDateEdit" name="deDogumTarihi"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Müşteri Numarası</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leMusteriNo"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnKaydet">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTemizle">
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnIptal">
       <property name="text">
        <string>İptal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnKaydet</sender>
   <signal>clicked()</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnIptal</sender>
   <signal>clicked()</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTCNo</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>DegisiklikVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAdSoyad</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>DegisiklikVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deDogumTarihi</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>DegisiklikVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pteAdres</sender>
   <signal>textChanged()</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>DegisiklikVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTelefon</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>DegisiklikVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leEposta</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>DegisiklikVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leInstagram</sender>
   <signal>textChanged(QString)</signal>
   <receiver>OOStandartMusteriBilgileriFormu</receiver>
   <slot>DegisiklikVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>DegisiklikVar()</slot>
 </slots>
</ui>
